views/login.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Century+Gothic&display=swap">
+    <title>Login | Kainos</title>
+    <link rel="icon" href="../public/images/favicon.ico">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link  rel="stylesheet" href="../public/css/styles.css">
+</head>
+<body>
+    <main class="flex-shrink-0">
+        <div class="spec-jobs-container container" id="spec-container"> 
+            <section class="vh-100 gradient-custom">
+                <div class="container py-5 h-100">
+                  <div class="row d-flex justify-content-center align-items-center h-100">
+                    <div class="col-12 col-md-8 col-lg-6 col-xl-5">
+                      <div class="card text-white shadow" style="border-radius: 1rem;">
+                        <div class="card-body p-5 text-center">
+                          <div class="mb-md-5 mt-md-4 pb-5">
+                            <div class="mb-5">
+                                <img src="../public/images/Kainos-alt-transparent.png" alt="Kainos Logo" width="220" height="70">
+                            </div>
+                            <h2 class="fw-bold mb-2 text-uppercase">Login</h2>
+                            <p class="text-white-50 pt-2 mb-5">Please enter your email and password!</p>
+                              <div class="container">
+                                {% if errormessage %}
+                                  <div id="login-error" class="alert alert-danger" role="alert">
+                                    {{ errormessage }}
+                                  </div>
+                                {% endif %}
+                              </div>
+                              <form action="/login" method="post">
+                                <div class="form-outline form-white mb-5">
+                                  <input type="email" id="username" name="username" class="form-control form-control-lg shadow-sm" placeholder="Email" value="{{ username }}" />
```

Comment:
is it email or username

Reply:
Yeah that makes sense, although it probably would have been best to call this email consistently. Don't worry about it  :)